pkgs/development/python-modules/thefuzz/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "thefuzz";
+  version = "0.19.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-b3Em2y8silQhKwXjp0DkX0KRxJfXXSB1Fyj2Nbt0qj0=";
+  };
+
+  propagatedBuildInputs = [ python-Levenshtein ];
```

Comment:
`python-levenshtein` is an [extra dependency](https://github.com/seatgeek/thefuzz/blob/master/setup.py#L26).

```
  passthru.optional-dependencies = {
    speedup = [
      python-levenshtein
    ];
  };
```

Reply:
how would one enable this optional dependency when installing the package?

Reply:
@sumnerevans, see section 17.27.2.3.2 in the [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#reference).

@fabaff python-levenshtein (a C extension) seems to work on all platforms that support Python. Is there any downside to enabling it by default?

Reply:
I think that `thefuzz` should work the same way as `fuzzywuzzy and if most user would enable it anyway then it makes perfect sense to enable it by default.